tasks/fix-cat2.yml

```diff
@@ -1380,24 +1380,29 @@
   tags:
       - RHEL-07-030350
 
-- name: "MEDIUM | RHEL-07-030360 | All privileged function executions must be audited."
+- name: "MEDIUM | RHEL-07-030360 | PATCH | The Red Hat Enterprise Linux operating system must audit all executions of privileged functions."
   block:
-      - name: "MEDIUM | RHEL-07-030360 | AUDIT | All privileged function executions must be audited. (find suid/sgid programs)"
-        command: find {{ rhel7stig_local_mounts | join(' ') }} -xdev -type f ( -perm -4000 -o -perm -2000 )
-        check_mode: no
-        changed_when: no
-        register: rhel_07_030360_audit
+      - name: "Delete obsolete suid/sgid rules"
+        file:
+            path: "/etc/audit/rules.d/rhel7stig_suid_sgid_commands.rules"
+            state: absent
 
-      - name: "MEDIUM | RHEL-07-030360 | All privileged function executions must be audited."
+      - name: "Audit rules for execve calls"
```

Comment:
I think we want to keep the name the same it helps with mapping output to specific rules

Reply:
Changed names of block tasks, see latest commit, what do you think about this ? Not sure if you like the extra comments in the block names (Delete obsolete suid/sgid rules) and (Audit rules for execve calls)